FreeRTOS/FreeRTOS-Kernel
# Add an example of human readable table generated by vTaskListTasks() to tash.h

Description
-----------
The task table returned by vTaskListTasks is human readable. But it misses a description for each column.
![image](https://github.com/user-attachments/assets/ee7fb61f-6c1f-4cca-a1a7-a22dae5d737a)
Add an sample task table to vTaskListTasks() comment in task.h. 
![image](https://github.com/user-attachments/assets/64fe60ca-926b-45f4-9993-2ab80118e41b)

Test Steps
-----------
NA

Checklist:
----------


- [Yes ] I have tested my changes. No regression in existing tests.
- [Yes ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS-Kernel/issues/1213

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
